### ISSUE TITLE
Pretrained model for the VIP dataset

### ISSUE THREAD

**Issue by sicxu** [xvjiarui/VFS#14]
Thanks for the code !!!
Could you also release the pretrained model for the VIP dataset? That would be very helpful!

**Comment by xvjiarui** [comment]
Hi @sicxu 
The model doesn't need to be fine-tuned for specific dataset for fine-grained correspondence. You may directly use it to inference.

**Comment by sicxu** [comment]
Cool, I will check the paper details later and thanks for the reply.

**Comment by sicxu** [comment]
Hi, @xvjiarui , I have read the paper and I am curious about the evaluation for the VIP dataset, **could you share the evaluation code for this dataset?** 

Something is not clear here, I hope you can help to answer my following concerns.
In your paper, not the only the first gt mask but also the prediction results in the latest **20** frames are propagated to the current frame. It seems that previous methods only use the first gt mask with several  (**1** in _Joint-task Self-supervised Learning for Temporal Correspondence_ ) prediction results to do the propagation, is there any ablation study about this number?

**Comment by xvjiarui** [comment]
Hi @sicxu 
You may use VIP dataset [here](https://github.com/xvjiarui/VFS/blob/master/mmaction/datasets/vip_dataset.py) for testing. 
I may include the scrip in the future.

**Comment by qianduoduolr** [comment]
hi, do you have the plan to include the evaluation script for VIP and JHMDB ? Looking forward to your reply.
